src/BloomExe/Edit/WebThumbNailList.cs

```diff
@@ -291,6 +300,23 @@ private void ItemClick(string s)
 				InvokePageSelectedChanged(page);
 		}
 
+		private void MenuClick(string pageId)
+		{
+			IPage page;
+			var menu = new ContextMenuStrip();
+			if (!_pageMap.TryGetValue(pageId, out page))
+				return;
+			foreach (var item in ContextMenuItems)
+			{
+				var useItem = item; // for use in Click action (reference to loop variable has unpredictable results)
```

Comment:
I'm finding "useItem", item, "Item1", "Item3" 100% inscrutable. Can't really have a more content-free set of names. But this is code doing a simple thing, so I don't think it's worth agonizing over.